operator/roles/kiali-deploy/defaults/main.yml

```
@@ -127,9 +127,20 @@ kiali_defaults:
 
   kubernetes_config:
     burst: 200
-    qps: 175
     cache_enabled: false
+    cache_istio_types:
+    - "VirtualService"
+    - "DestinationRule"
+    - "Gateway"
+    - "ServiceEntry"
     cache_duration: 300000000
+    excluded_workloads:
```

Make sure to specify default values (even if empty) for all settings. Only in rare cases do we not define values in here. This ensures the values get into the configmap and are not left undefined (which could cause errors in the operator ansible scripts or the kiali server itself).
```suggestion
    cache_namespaces: []
    excluded_workloads:
```